src/sentry/models/organization.py

```diff
@@ -232,14 +234,27 @@ def merge_to(from_org, to_org):
                     slug=project.slug,
                 )
 
-        # TODO(jess): figure out how to merge Releases and related models
-        # once we add organization, version unique constraint
-        for model in (ApiKey, AuditLogEntry, Release, ReleaseFile):
+        # TODO(jess): update this when adding unique constraint
+        # on version, organization for releases
+        for release in Release.objects.filter(organization=from_org):
+            try:
+                to_release = Release.objects.get(
+                    version=release.version,
+                    organization=to_org
+                )
+            except Release.DoesNotExist:
+                Release.objects.filter(
```

Comment:
it seems like since this isn't used that often and i'll be changing it once this goes out and we can add a unique constraint this will probably be fine, but let me know if you think we should use a lock here.